creed/tiling_utils_test_cases.py

```diff
@@ -0,0 +1,214 @@
+import numpy as np
+
+from creed.tiling_utils import XYCoord
+
+# for tiled region param dict creation
+_PARAM_SET_MOLY_INTERVAL_VALUE_CASES = [0, 1]
+
+
+# for tiled region FOV generation
+class TiledRegionRandomizeCases:
```

Comment:
Putting something in the cases file is useful to hide complicated functions. For simple lists of values, having them in the decorator is better, because than its easy to see what's being tested